app/components/candidate_interface/application_form_course_choices_component.rb

```diff
@@ -1,11 +1,12 @@
 module CandidateInterface
   class ApplicationFormCourseChoicesComponent < ViewComponent::Base
-    def initialize(choices_are_present:, completed:)
+    def initialize(choices_are_present:, completed:, number_of_choices:)
       @choices_are_present = choices_are_present
       @completed = completed
+      @number_of_choices = number_of_choices
     end
 
-    attr_reader :choices_are_present, :completed
+    attr_reader :choices_are_present, :completed, :number_of_choices
```

Comment:
Minor refactoring suggestion: once we have `number_of_choices` the `choices_are_present` attribute could be removed and `choices_are_present?` could return `number_of_choices > 0`?

Reply:
Good idea too - thanks, done ♻️